docs/source/analyze/cluster-resource.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Cluster Resource
+description: Analyze any attribute collected by the `ClusterResources` Collector
+---
+
+The `clusterResource` analyzer can be used to check any attribute of any known resources in the cluster in a generic manner.
+
+The `clusterResource` analyzer uses data from the [clusterResources collector](/collect/cluster-resources).
+The `clusterResources` collector is automatically added and will always be present.
+
+You must specify the `kind` and `name` attributes, plus the optional `namespace` attribute to target the Kubernetes resource.
+
+The `yamlPath` attribute is used to specify a dot-delimited YAML path of a property on the Kubernetes resource referenced in `name`.
+The `when` attribute supports standard arithmetic comparison operators.
+
+The outcomes on this analyzer will be processed in order, and execution will stop after the first outcome that is truthy.
+
+## Parameters
+
+**checkName**: (Optional) Analyzer name.
+Used for uniqueness if multiple analyzers are defined with similar parameters.
+
+**kind**: (Required) The type of Kubernetes resource being targeted by `name`.
+
+**name**: (Required) The name of the resource to check.
+
+**namespace**: (Optional) The namespace to look for the resource in.
+
+**yamlPath**: (Required) The dot-delimited YAML path of a property on the Kubernetes resource.
+
+**regex**: (Optional) See [Regular Expression](/analyze/regex) documentation.
+Note: when using arithmetic comparison in `when`, this is not supported at this time.
+
+**regexGroups**: (Optional) See [Regular Expression](/analyze/regex) documentation.
+Note: when using arithmetic comparison in `when`, this is not supported at this time.
+
+## Outcomes
+
+The `when` value in an outcome of this analyzer can accept a few variations.
+
+If the `yamlPath` points to a quantity-based value (eg. the size or quota or something), standard arithmetic comparison operators may be used.
+eg. `<`, `<=`, `>`, `>=`, `==`, `!=`. 
+
+Alternatively if `regex` specifies an expected value, using a boolean `true` or `false` in the `when` clause is acceptable.
```

Comment:
in the yaml, we are using the string true and false, should we need to clarify here or support both?
```
- pass:
     when: "true"  or  true ?
     message: is ReadWriteOnce
```

Reply:
fixed in https://github.com/replicatedhq/troubleshoot.sh/pull/503/commits/e80d0345093c8bf0732a6785d100fd2b10ffaf08